app/assets/stylesheets/grader.scss

```diff
@@ -238,7 +238,7 @@ ul.annotation_text_list {
   width: 300px;
 
   li {
-    @include border-radius-top(0 !important);
+    border-radius: 0 !important;
```

Comment:
!important should not be used<br>Properties should be ordered background, border-radius, padding, width